src/components/calculator.js

```diff
@@ -1,9 +1,132 @@
-import React from 'react';
```

Comment:
![Screenshot from 2023-06-08 17-31-02](https://github.com/wolterbwambale/math-magician-app/assets/83042742/2e495a84-01ac-4890-9cb4-2528ad6a863c)
- [ ] `README.md` file: To make your README more descriptive and professional, I think it would be better to update your `Getting Started` section with correct and appropriate information to each sub-section such as information of how `clone` your application, `access the cloned` directory, `install` dependencies,  `run build` for production, `use/run` your application, `run tests`(if exist),... **To help you fill the above information, you can just refer [this Gist file](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96).**